coffee-shop-application/src/main/kotlin/com/motycka/edu/order/OrderDTO.kt

```diff
@@ -2,7 +2,17 @@ package com.motycka.edu.order
 
 
 data class OrderDTO(
-    val id: OrderId?,
+    val id: Long?,
```

Comment:
There are practical reasons why I tend to use typealias for ids ...
1. I can change data types golbaly if needed (for example from Int to Long)
2. I can search all references in code ... for example it is hard to search for all occurrences of Long that refer to order ID but it is easy to search for all occurrences of OrderId